include/coreir/ir/common.h

```diff
@@ -18,6 +18,9 @@ typedef std::set<Connection, ConnectionCompFast> ConnectionsFast;
 bool isNumber(std::string s);
 bool isPower2(uint n);
 
+bool isSlice(std::string selstr);
+void parseSlice(std::string selstr, uint* low, uint* high);
```

Comment:
```suggestion
std::pair<int, int> parseSlice(const std::string& selstr);
```

 (may need to `#include <pair>` at the top)